### DISCUSSION TITLE
Minecraft in v86

### DISCUSSION THREAD

**Discussion by ayunami2000** [copy/v86#672] [General]
![MC IN BROWSER POG](https://user-images.githubusercontent.com/24785136/112726725-4fb00700-8ef5-11eb-97e0-da1d28280c2c.PNG)
I did this a few months ago. It takes a while and after taking the screenshot I noticed it had frozen. But IT WORKS YOOOOOOOOOOOOOOOO!!!

I have an intuitive state file saved to launch the game in arch linux with instructions to fix the network. I will not be sharing it as its a large file and slow and also contains a version of minecraft on it which makes it possibly illegal to upload. (it's an old beta version so I doubt it would matter if I uploaded it anyways)

I chose the older version just because it's a faster download, the networking is slow enough as is so the latest version would take suuuuper long.

Credit [portablemc](https://github.com/mindstorm38/portablemc) for launching the game and ME for the intuitive yet still **unreleased** UI for choosing the version and launching the game all in one go.

Also side note: chrome in firefox [(slax linux)](http://slax.org)
![CHROME IN FIREFOX](https://user-images.githubusercontent.com/24785136/112726890-0f04bd80-8ef6-11eb-8fbd-fa3a61b5a762.PNG)

**Comment by hello-smile6** [comment]
Can I have your iso for Chrome?

**Comment by hello-smile6** [comment]
I can't get it to work.

**Comment by ayunami2000** [comment]
> 
> 
> I can't get it to work.

it took forever to get to work so chances are you wont get it to work.

> 
> 
> Can I have your iso for Chrome?

for chrome, use the 32-bit ISO from http://slax.org

**Comment by ayunami2000** [comment]
Update: I was able to get in game in Minecraft b1.7.3 (chosen because no animated title screen) and I was able to create a working savestate! Although it runs slowly, it actually runs at 1 frame every 2-5 seconds which is NOT BAD considering what it is. (walking is difficult though and usually does not go through. same with breaking.)
![copymc_INGAME](https://user-images.githubusercontent.com/24785136/116702247-dba2cc00-a996-11eb-9239-400e93131c6d.PNG)
![copymc_INGAME2](https://user-images.githubusercontent.com/24785136/116702258-df365300-a996-11eb-93b4-65932c476f79.PNG)
![copymc_INGAME3](https://user-images.githubusercontent.com/24785136/116702262-e1001680-a996-11eb-9743-fccf18343d82.PNG)
![copymc_INGAME4](https://user-images.githubusercontent.com/24785136/116702272-e3fb0700-a996-11eb-9953-3b282bb02b4f.PNG)

*the yellow outline is there because I streamed it on Twitch out of boredom

edit: here's a quick video of it https://www.youtube.com/watch?v=B5P3vVjJoz8

**Comment by prozacgod** [comment]
I'm dying right now, but.... DO unreal tournament 99 (native linux) next, I keep wanting to try it out myself, but too many projects!   I figure it's software renderer was fairly good for it's era, so... maybe it'd play at 2 fps?

**Comment by d1y** [comment]
this is fun

**Comment by THESTORMZ** [comment]
![image](https://user-images.githubusercontent.com/74722466/132002541-a51b63df-ae49-4d0f-bc2e-2ffb9bbb7c7e.png)
i get this its frozen

**Comment by hello-smile6** [comment]
> ![image](https://user-images.githubusercontent.com/74722466/132002541-a51b63df-ae49-4d0f-bc2e-2ffb9bbb7c7e.png) i get this its frozen

It works about a minute after. Try https://copy.sh/v86/?profile=archlinux&c=.%2Fnetworking.sh%3B.%2Fstartx.sh%26echo%20'sleep%20240%3Bxterm'%3E.%2Fxterm-launch.sh%3B.%2Fxterm-launch.sh%26 (Will enable networking and start GUI, then open xterm)

**Comment by THESTORMZ** [comment]
now this 
![Screenshot 2022-01-10 8 42 26 AM](https://user-images.githubusercontent.com/74722466/148775839-720b82e2-36ba-483c-bc48-53af1b5ec106.png)

**Comment by THESTORMZ** [comment]
can you give the savestate to me

**Comment by ProgrammerIn-wonderland** [comment]
Maybe one day if there's GPU acceleration you might get 2 fps

**Comment by ISSSSS212** [comment]
its the eaglercraft guy!

**Comment by elletsonchase-lang** [comment]
uh what? this cant be illegal becuase notch is long gone heck you can literly play minecraft xbox360 version the full game for free why THEY ABANDONED THOSE OLD GAMES Microsoft is now head of Mojang so there are many free minecraft versions and alternatives
